hazelcast/src/test/java/com/hazelcast/internal/util/phonehome/PhoneHomeTest.java

```diff
@@ -503,5 +506,93 @@ public void testFlakeIDGeneratorCount() {
+
+        IMap<Object, Object> iMap = node.hazelcastInstance.getMap("hazelcast");
+        LocalMapStatsImpl localMapStats = (LocalMapStatsImpl) iMap.getLocalMapStats();
+
+        localMapStats.incrementPutLatencyNanos(2000000000L);
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("mpptla"), String.valueOf(2000));
+
+        localMapStats.incrementPutLatencyNanos(1000000000L);
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("mpptla"), String.valueOf(1500));
+
+        localMapStats.incrementPutLatencyNanos(2000000000L);
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("mpptla"), String.valueOf(1666));
+
+    }
+
+    @Test
+    public void testMapGetLatencyWithoutMapStore() {
+        Map<String, String> parameters;
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("mpgtla"), "-1");
+
+        IMap<Object, Object> iMap = node.hazelcastInstance.getMap("hazelcast");
+        LocalMapStatsImpl localMapStats = (LocalMapStatsImpl) iMap.getLocalMapStats();
+
+        localMapStats.incrementGetLatencyNanos(2000000000L);
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("mpgtla"), String.valueOf(2000));
+
+        localMapStats.incrementGetLatencyNanos(2000000000L);
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("mpgtla"), String.valueOf(2000));
+    }
+
+    void initialiseForMapStore(String mapName) {
+        MapStoreConfig mapStoreConfig = new MapStoreConfig();
+        mapStoreConfig.setEnabled(true);
+        mapStoreConfig.setImplementation(new DelayMapStore());
+        node.getConfig().getMapConfig(mapName).setMapStoreConfig(mapStoreConfig);
+    }
+
+    @Test
+    public void testMapPutLatencyWithMapStore() {
+        Map<String, String> parameters;
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("mpptlams"), "-1");
+
+        IMap<Object, Object> iMap1 = node.hazelcastInstance.getMap("hazelcast");
+        initialiseForMapStore(iMap1.getName());
+        iMap1.put("key1", "hazelcast");
+        iMap1.put("key2", "phonehome");
+        parameters = phoneHome.phoneHome(true);
+        assertGreaterOrEquals("mpptlams", Long.parseLong(parameters.get("mpptlams")), 200);
+
+        IMap<Object, Object> iMap2 = node.hazelcastInstance.getMap("phonehome");
+        initialiseForMapStore(iMap2.getName());
+        iMap2.put("key3", "hazelcast");
+        parameters = phoneHome.phoneHome(true);
+        assertGreaterOrEquals("mpptlams", Long.parseLong(parameters.get("mpptlams")), 200);
+    }
+
+
+    @Test
+    public void testMapGetLatencyWithMapStore() {
+        Map<String, String> parameters;
+        parameters = phoneHome.phoneHome(true);
+        assertEquals(parameters.get("mpgtlams"), "-1");
+
+        IMap<Object, Object> iMap1 = node.hazelcastInstance.getMap("hazelcast");
```

Comment:
You can create the map in the `initialiseForMapStore()` method and return the map. That would reduce the 2-line initalisations to 1 line (in all above tests).